docs/learn-nominator.md

```diff
@@ -99,6 +87,20 @@ validator's nominators. Any account can do this, although in practice validator
 this as a service to their nominators. See the page on [Simple Payouts](learn-simple-payouts) for
 more information and instructions for claiming rewards.
 
+
+#### Required Minimum Stake
+
+Due to the way the [Phragmen algorithm](learn-phragmen) generates the solution set, there was
+**previously** a minimum number of DOT needed to nominate with in order to receive staking
+rewards due to the fact that the solution set need to be both proccessed and stored within
+a single block, 6 seconds on Polkadot.
+
+A [runtime upgrade](build-protocol-info#runtime-upgrades) moved the logic to process the rewards
+for nominators to an [off-chain worker](learn-phragmen#off-chain-phragmén), and now the minumum
+DOT required is only limited by the [existential deposit](build-protocol-info#existential-deposit)
+See the [Election Solution Set](learn-nominator#the-election-solution-set) section on the
+Nominator page for more details.
```

Comment:
```suggestion
DOT required is only limited by the [existential deposit](build-protocol-info#existential-deposit).
Note that this might change if enough new nominators nominate. In that case, the minimum can 
rise above the ED again. This is unlikely to happen in the next 6 to 12 months.

See the [Election Solution Set](learn-nominator#the-election-solution-set) section on the
Nominator page for more details.
```

Reply:
Something like this should do?

Reply:
I am generally against any reference to the present in specific terms like " next 6 to 12 months" - from when? Readers don't know when and this doc (likely) will go stale before it's corrected if this happens.

Reply:
Agreed, but "in 2021" felt strange and leaving it out makes things sound less safe than they are, leaving readers with uncertainty. Ideas?

Reply:
How about "Although at the time of writing, with the number of nominators at ~ 25_000, there is no minimum, if the number of nominators increases dramatically in the future, the solution will be pruned and nominators with the least amount of stake will once again be pruned and receive no staking rewards."

Reply:
I have updated the text, take a look at the changes that sum up what you suggest without referencing a specific time @laboon 🙏🏼

Reply:
lgtm